_posts/2020-11-09-ten-tips-for-tidy-data.markdown

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Awesome post! I wish someone told me about this when I started producing a ton of data and soon got lost in all my spreadsheets and results ;)

Reply:
Thanks! Same here!